examples/td_heat_conduction/triZone-oneDynamicDT-Plate/Data/readme.md

```diff
@@ -0,0 +1 @@
+This directory contains files and data related to the varying parameter.
```

Comment:
since this reamd in inside a specific case, can the varying parameter be described with more details?